pkg/grid/CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing to Landscape
+
+Thank you for your interest in contributing to the Urbit ecosystem.
+
+Landscape is entirely open to contributions from the community. We mainly organize through our [project board], [issues], and [weekly call].
+
+For now our code is stored in the main [urbit repo]. If you would like to contribute feel free to open up a PR there.
+
+## Git Conventions
+
+For Landscape we follow the same conventions as the main repo which can be found in it's [contributing doc]. This can be summarized as the following:
+
+Commits should try to be atomic and focused on one feature at a time. Work-in-progress commits should be rebased and combined into one so that the commit history stays clean.
+
+Commits should follow this format:
+
+> component: short description
+>
+> long description
+
+Where `component` is the closest most relevant area of the code base written as concisely as possible. The short description that accompanies should be a super concise summary of the changes. The total length of the commit message should strive to be 50 characters or less. The long description is optional, but should be used if further explanation is necessary.
+
+### Pull Requests
+
+A pull request (PR) should have a title similar in structure to our commit messages where it has a short identifier component followed by a very concise summary of the PR's intent. All PRs should have a description further laying out what it accomplishes. If the PR addresses certain Github issues, those should be referenced in the body of the description so they get linked. If there are visual or design changes screenshots should also be included.
+
+PRs to this repo should currently ask for review from the following contributors:
```

Comment:
Developers who aren't a member of the urbit organisation cannot ask for a review iirc, they should just tag the relevant contributor in a cc: